### PR TITLE
Added the rendering of maps

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

File: src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacer.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        } catch (ContentReadException | MapRendererException e) {
+            Log.build("Failed rendering map, uri: " + figureUri + ", error: " + e, Level.ERROR).log();
+            return matcher.group();
+        }
+    }
+
+    private String invokeMapRenderer(String postBody) throws MapRendererException {
+        try (CloseableHttpResponse response = httpClient.sendPost(rendererPath, HEADERS, postBody, CHARSET)){
+            return IOUtils.toString(response.getEntity().getContent());
+        } catch (IOException e) {
```
Comment (daiLlew) [inline src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacer.java]: Log here please.

File: src/test/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacerTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    public void replaceShouldGetContentAndInvokeMapRendererForSvg() throws Exception {
+        String json = "{\"foo\": \"bar\"}";
+        when(contentResponseMock.getAsString()).thenReturn(json);
+        when(httpClientMock.sendPost(Configuration.MAP_RENDERER.getSvgPath(), null, json)).thenReturn(responseMock);
+        when(responseMock.getEntity().getContent()).thenReturn(IOUtils.toInputStream(mapHtml));
+        when(templateServiceMock.renderTemplate(template, singletonMap("foo", "bar"), singletonMap("mapHtml", mapHtml))).thenReturn(renderedTemplate);
+
+        String result = testObj.replace(matcher);
+
+        assertThat(result, equalTo(renderedTemplate));
```
Comment (daiLlew) [inline src/test/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacerTest.java]: General comment on each of these test cases:  can you add `verify(...)` checks to ensure the mocks are being called the expected number of times with the expected args - http client, template service

File: src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacer.java (hunk 1 above)
Comment (YetAnotherMatt) [reply src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacer.java]: that exception is caught and logged in the replace method, line 99:
```
        } catch (ContentReadException | MapRendererException e) {
```

File: src/test/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacerTest.java (hunk 2 above)
Comment (YetAnotherMatt) [reply src/test/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/MapTagReplacerTest.java]: done

**Pull request merged**